db/migrate/20180109222722_create_user_second_factors.rb

```diff
@@ -0,0 +1,12 @@
+class CreateUserSecondFactors < ActiveRecord::Migration[5.1]
+  def change
+    create_table :user_second_factors do |t|
+      t.integer :user_id, null: false
+      t.string :method, null: false
```

Comment:
@featheredtoast Is there a reason we need this `method` column as I don't see us using it anywhere.